src/components/Breadcrumbs/CurrentBreadcrumbItem.tsx

```diff
@@ -0,0 +1,50 @@
+import { Badge, BadgeProps } from "@elements/Badge/Badge";
+import { merge } from "@utilities/merge";
+import React, { FC } from "react";
+import { Breadcrumb } from "./Breadcrumbs";
+
+const ItemWithBadges: FC<{ badges?: BadgeProps[] }> = ({ badges, children }) => (
+    <span className="tw-inline-flex tw-gap-x-2 tw-items-center">
+        {children}
+
+        {badges?.map(({ children: badge, ...props }, index) => (
+            <Badge {...props} key={`breadcrumb-badge-${index}`}>
+                {badge}
+            </Badge>
+        ))}
+    </span>
+);
+
+export const CurrentBreadcrumbItem: FC<Breadcrumb> = ({ label, badges, bold, decorator, link, onClick }) => {
+    const classNames = merge([decorator && "tw-flex tw-gap-x-1 tw-items-center", bold && "tw-font-bold"]);
```

Comment:
Looking good :+1: Only minor thing: the current breadcrumb item has `tw-text-black` and `dark:tw-text-white` seems not to be applied.